### PR TITLE
independent challenge

### PR DESCRIPTION
**Pull request by joeray100**

### CODE REVIEW

**Comment by joeray100** [comment]
Joe Ray - Apologies I forgot to put my name in the first time around!

File: lib/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    @dishes = []
+  end
+
+  def closing_time(time)
+    if @name =='Fuel Cafe'
+      '18:00'
+    elsif @name == 'Il Poggio'
+      '23:00'
+    end
+  end
```
Comment (memcmahon) [inline lib/restaurant.rb:18]: What if we live in a world with more than two restaurants?

We wanted to see you do some string/integer conversion and manipulation for this one.  Generally, when you see that a method takes an argument, you will use that argument in the method somewhere.

File: lib/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    @dishes << dish
+  end
+
+  def open_for_lunch?
+    if @opening_time < '12:00'
+      true
+    else
+      false
+    end
+  end
```
Comment (memcmahon) [inline lib/restaurant.rb:30]: This works, but you could also rely more on the return value of the conditional:

```ruby
def open_for_lunch?
  @opening_time < '12:00'
end
```

File: lib/restaurant.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    @dishes.map(&:upcase)
+  end
+
+  def announce_closing_time(time)
+    if @name =='Fuel Cafe'
+      "Fuel Cafe will be closing at 11:00AM"
+    elsif @name == 'Il Posto'
+      "Il Posto will be closing at 11:00PM"
+    end
+  end
```
Comment (memcmahon) [inline lib/restaurant.rb:42]: Again, I want to live in a world with more than two restaurants....

File: test/restaurant_test_1.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/restaurant'
+
+class RestaurantTest < Minitest::Test
```
Comment (memcmahon) [inline test/restaurant_test_1.rb:5]: These tests look good, but you should keep all Restaurant tests in one file

**Review by memcmahon — COMMENTED** [review]
Good start - take a look at my notes and let me know if you have any questions.

Score: 2.5

**Pull request closed without merging**